kythe/go/extractors/config/smoke/smoke.go

```diff
@@ -187,7 +187,7 @@ func (g harness) filenamesFromExtraction(repoURI string) (map[string]bool, error
 	ret := map[string]bool{}
 	err = filepath.Walk(tmpOutDir, func(path string, info os.FileInfo, err error) error {
 		if err == nil && filepath.Ext(path) == ".kindex" {
-			cu, err := kindex.Open(context.Background(), path)
```

Comment:
(as a side effect we fix this problematic context.Background() I stuffed in there before too)